evm_interpreter/src/instructions/environment.rs

```diff
@@ -33,12 +33,15 @@ impl<S: EthereumLikeTypes> Interpreter<'_, S> {
         Ok(())
     }
 
-    pub fn difficulty(&mut self, system: &mut System<S>) -> InstructionResult {
+    pub fn difficulty(&mut self, _system: &mut System<S>) -> InstructionResult {
         self.gas
             .spend_gas_and_native(gas_constants::BASE, DIFFICULTY_NATIVE_COST)?;
+        #[cfg(feature = "prevrandao")]
```

Comment:
Why not add feature for value in block context generally? So difficulty will be aligned with value in block context/header?